code/datums/supply_packs/_supply_packs.dm

```diff
@@ -25,6 +25,13 @@ var/list/all_supply_groups = list(
 	"Reagent tanks",
 )
 
+var/list/contraband_supply_groups = list(
```

Comment:
could define this as static on /obj/structure/machinery/computer/supplycomp

Reply:
wahhhhhhhhhh